bbohling22/gitignore-setup
# Superfeature

### Testing instructions
1. open your IDE, open README.md
2. Notice added descriptions under header "Demo-repo"
